server/pulp/plugins/importer.py

```diff
@@ -8,6 +16,29 @@ class Importer(object):
     importer during discovery.
     """
 
+    @staticmethod
+    def get_downloader(config, url, **options):
+        """
+        Get a configured downloader.
+
+        :param config: A plugin configuration.
+        :type config: pulp.plugins.config.PluginCallConfiguration
+        :param url: A URL.
+        :type url: str
+        :param options: Extended configuration.
+        :type options: dict
+        :return: A configured downloader.
+        :rtype: nectar.downloaders.base.Downloader
+        :raise ValueError: when the URL scheme is not supported.
+        """
+        url = urlparse(url)
+        nectar_config = importer_config_to_nectar_config(config.flatten())
+        if url.scheme == 'file':
```

Comment:
Here and 2 lines below should call ``.lower()`` on the scheme to make the comparison case-insensitive.